### DISCUSSION TITLE
HTTP or MQTT

### DISCUSSION THREAD

**Discussion by Bronnnn** [mindsphere/node-red-contrib-mindconnect#157] [General]
I am just curious how the Agent sends data to MindSphere. 
In the image that you added it says HTTP requests, so is it correct that it does not use MQTT?

Its probably a trivial question, but i couldnt find an answer in the documentations or with wireshark, with my limited experience ._.

**Comment by sn0wcat** [comment]
@Bronnnn it is correct, the node is using HTTP calls to mindconnect api, there is no MQTT involved in this version of the node (yet).
